integration-tests/logging-panache/src/test/java/io/quarkus/logging/GenerateAllLogUsages.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.logging;
+
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Parameter;
+import java.util.Arrays;
+import java.util.Comparator;
+
+import org.jboss.logging.Logger;
+
+public class GenerateAllLogUsages {
```

Comment:
Why this is not a test class?

Reply:
Because it's not? It generates a bean with all possible `Log.*` invocations, which is then used in a test.